### PR TITLE
Fix JDK version to 1.8 and update enforcer message

### PR DESCRIPTION
**Pull request by pzygielo**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -72,8 +72,8 @@
                         <configuration>
                             <rules>
                                 <requireJavaVersion>
-                                    <version>[1.8.0,1.9.0)</version>
-                                    <message>You need JDK8 or lower</message>
+                                    <version>1.8.0</version>
+                                    <message>You need JDK8 or higher</message>
```
Comment (bshannon) [inline pom.xml]: I think this change would limit it to 1.8.0 and not allow 1.8.1,
not that I ever expect there to be a 1.8.1.  As currently written,
it allows 1.8.*.  "or lower" is probably wrong.

Comment (pzygielo) [reply pom.xml]: https://maven.apache.org/enforcer/enforcer-rules/versionRanges.html:
> 1.0 | x >= 1.0 * The default Maven meaning for 1.0 is  everything (,) but with 1.0 recommended. Obviously this doesn't work for  enforcing versions here, so it has been redefined as a minimum version.

**Comment by pzygielo** [comment]
>  As currently written, it allows 1.8.*.

So I left the rule unchanged and updated requirement and message printed if rule is violated.

File: pom.xml (hunk 1 above)
Comment (bshannon) [reply pom.xml]: Sorry, haven't had time to get back to this to verify that the fix works as expected.

Comment (pzygielo) [reply pom.xml]: There are still two commits, 1<sup>st</sup> one assuming wrong implementation in pom, and 2<sup>nd</sup> assuming wrong requirement with correctly expressed rule. I don't know which one is right, so feel free to merge whichever you prefer or none.

**Review by bshannon — APPROVED** [review]

**Pull request merged**